styles/groups.less

```diff
@@ -24,6 +24,18 @@
     }
   }
 
+  > .bpp-checkbox {
```

Comment:
I think we should add this to the `properties.less`, since it's more properties than groups related.

Reply:
Done